tests/js/client/shell/shell-cluster-collection.js

```diff
@@ -989,6 +989,47 @@ function ClusterCollectionSuite () {
       }
     },
 
+    // regression test for https://github.com/arangodb/arangodb/pull/21071
+    testCreateWithSlowCurrentUpdate : function () {
+      let setFailAt;
+      let removeFailAt;
+      if (isServer) {
```

Comment:
`isServer` should always be false here, as the test file is located in `tests/js/client`.